Assignment1/Assignment1_QNo1/SolAssignmentNo1_QNo1.rb

```diff
@@ -0,0 +1,22 @@
+# => 1. Find out how to convert the string "7.13" into the number 7.13
+# => author : Hrisikesh Singha Chowdhury
+
+# => Fn to implement buisness logic
+def cvt_str_to_int
+  puts "Enter another string"
+  str = gets.chomp
+  str_output = str.chomp.to_f
+
+  if (str.nil? && str_output == 0)
```

Comment:
@HrisikeshSinghaChowdhury There is a inbuilt method `zero?` in ruby , Please use that

Reply:
ok